updated dockerfiles to build with rust 1.64, added dependency fontconfig-dev

spital opened the pull request.

spital commented:
I'll update deps_setup.sh but I never did any squash/merge/rebase. Can you 
give me a lead please?
 my new local branch rust164 (pushed to github.com/spital/darkfi too). What 
now ?

---------- Původní e-mail ----------
Od: aggstam ***@***.***>
Komu: darkrenaissance/darkfi ***@***.***>
Kopie: spital ***@***.***>, Author ***@***.***>
Datum: 6. 10. 2022 14:29:55
Předmět: Re: [darkrenaissance/darkfi] updated dockerfiles to build with rust
1.64, added dependency fontconfig-dev (PR #110)
"


Hello spital!
Could you please squash affected files to a single commit?
Additionally, since you find package name differences, you can add them to 
our dependencies setup script, similar to #107
(https://github.com/darkrenaissance/darkfi/pull/107)!

—
Reply to this email directly, view it on GitHub
(https://github.com/darkrenaissance/darkfi/pull/110#issuecomment-1269952981)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/ACUF5GEX3HSMXUXHPP2I5OTWB3A35ANCNFSM6AAAAAAQ3PB6NM)
.
You are receiving this because you authored the thread. Message ID: <
***@***.***>
"

aggstam commented:
You can do that by using pull --rebase after committing your changes.
That will pull everything new and keep your single commit to push!
No worries tho, we already update the docker files you pushed!
I'm mentioning it for future references.
We can close this PR and you create a new one with the updated depencencies script.
Cheers!

spital commented:
Still no idea. I deleted my github branch, cloned it fresh.
* [new branch] fix/update_dependencies -> fix/update_dependencies
Will check and when ready, merge to my main and send PR.
Hope this helps.
Have a nice Friday!
---------- Původní e-mail ----------
Od: aggstam ***@***.***>
Komu: darkrenaissance/darkfi ***@***.***>
Kopie: spital ***@***.***>, Author ***@***.***>
Datum: 7. 10. 2022 9:49:13
Předmět: Re: [darkrenaissance/darkfi] updated dockerfiles to build with rust
1.64, added dependency fontconfig-dev (PR #110)
"


You can do that by using pull --rebase after committing your changes.
That will pull everything new and keep your single commit to push!
No worries tho, we already update the docker files you pushed!
I'm mentioning it for future references.
We can close this PR and you create a new one with the updated depencencies 
script.
Cheers!

—
Reply to this email directly, view it on GitHub
(https://github.com/darkrenaissance/darkfi/pull/110#issuecomment-1271239242)
, or unsubscribe
(https://github.com/notifications/unsubscribe-auth/ACUF5GAOX5WHW7373BJYQELWB7IXNANCNFSM6AAAAAAQ3PB6NM)
.
You are receiving this because you authored the thread. Message ID: <
***@***.***>
"
